### ISSUE TITLE
User story - Account preferences

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#12]
### If applicable, add a related Epic.

Epic: #5 

### User Story

As a **user**, I can **choose update by account preferences** so that I can **have more control into how my profile is view and what information is shared**.

### Acceptance Criteria

- User should have a preferences page
- The preferences page should allow the user to update details such as profile visibility, whether to show their email or their profile image
- Preferences should be distinct from account settings

### Please ensure you've completed these tasks before marking the issue as done.

- [ ] Reviewed the user story for clarity and completeness.
- [ ] Identified and linked related issues or epics.
- [ ] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [ ] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [ ] Updated the issue with any relevant notes or feedback (to date).
- [ ] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [ ] Add a path for `user/<slug>/preferences`
- [ ] Create a UI for the different enabled preferences that allows user to update preferences together

**Comment by stephendawsondev** [comment]
Marking this as a 5 because what makes this difficult is having to create another UI, or else have multiple forms and the same page, which leads to issues.
